extensions/wikia/Hydralytics/css/hydralytics.scss

```diff
@@ -153,10 +161,11 @@ $green-navy: #002A32;
 
 #analytics_confidential {
 	display: inline-block;
-	border: 2px solid $content-box-color;
-	background-color: $content-box-color;
+	width: 100%;
```

Comment:
Let's remove `display: flex` property from `#analytics_confidential_header`. Then we can get rid of `display` and `width` properties from here.